qiskit_nature/problems/second_quantization/molecular/molecular_problem.py

```diff
@@ -22,9 +22,10 @@
 from .fermionic_op_builder import build_fermionic_op, build_ferm_op_from_ints
 from .integrals_calculators import calc_total_magnetization_ints
 from .integrals_calculators import calc_total_particle_num_ints
+from ..base_problem import BaseProblem
 
 
-class MolecularProblem:
+class MolecularProblem(BaseProblem):
     """Molecular Problem"""
 
     def __init__(self, fermionic_driver: FermionicDriver,
```

Comment:
In the same way there is FermionicDriver in here I wonder whether we ought to have FermionicTransformer and likewise BosonicTransformer (here it would be a List of Fermionic ones of course)

Reply:
Why is this field even called `fermionic_driver` rather than just `driver` as in `BaseProblem`? Sub-typing it should be fine